projects/core/src/occ/occ-models/occ-endpoints.model.ts

```diff
@@ -263,4 +263,6 @@ export interface OccEndpoints {
    * @member {string}
    */
   asmCustomerSearch?: string;
+  budget?: string;
```

Comment:
Could you add code comments as above, please?

Reply:
is it enough?